exercises/randomgenerator.js

```diff
@@ -1,20 +1,34 @@
 
 
 class RandomGenerator{
-	constructor(){ 
+	constructor(minValue, maxValue){
+		this.minValue = minValue;
+		this.maxValue = maxValue;
+		if (minValue === undefined || maxValue === undefined){
+			this.minValue = 1;
+			this.maxValue = 10;
+		}
+		this.randomNumber = this.generate();
```

Comment:
you weren't required to generate the random number when you get the range.

Reply:
The danger is this: if you change the range, but expect the old number to still be there, you would be wrong.